src/main/java/org/ossgang/commons/observable/Observables.java

```diff
@@ -78,4 +72,42 @@ private Observables() {
         }
         return combinedProperty;
     }
+
+    /**
+     * Produces an {@link ObservableValue} that on each update of any source {@link ObservableValue} publishes the
+     * result of the combiner applied with the latest values of the other inputs.
+     * The order of the input values of the combiner is the same as the order of the provided source {@link ObservableValue}s
+     *
+     * @param sources  the input {@link ObservableValue}s
+     * @param combiner the combining function that will produce the result
+     * @param <I>      the input type
+     * @param <O>      the output type
+     * @return an {@link ObservableValue} that on each update of any source publishes the result of the
+     * combiner applied with the latest values of the other inputs.
+     */
+    public static <I, O> ObservableValue<O> mergeLatest(List<ObservableValue<I>> sources,
```

Comment:
I would strongly vote to be consistent as much as possible with commen reactive stream APIs. I would suggest to pick one (e.g. FLux, as we use it the most for the moment?). Then I think we should try to be consistent with them in both, naming and method signatures - where it makes sense of course ;-)

This would be useful e.g. for people who know already flux, to also use our operators for Observable values ...

Having this said, I think this method would have to be called `combineLatest(..)`. `merge` (if I understand correctly), in reactive frameworks usually applies to sequential merge of the same types ...

Reply:
Further, I think the combineLatest methods should allow different types .....

So if you work with Lists, they would have to be of ObservableValue<?> ....

To make it typechecked, one would have to use signatures like in Flux:

```
<T1, T2, R> ObservableValue<R> combineLatest(ObservableValue<T1> in1, ObservableValue<T2> in2, Func2<T1,T2,R> combiner);

<T1, T2, T3, R> ObservableValue<R> combineLatest(ObservableValue<T1> in1, ObservableValue<T2> in2, ObservableValue<T3> in3, Func3<T1,T2,T3,R> combiner);
...
...
```

Reply:
I think it would be nice to have both options, similar-typed and mixed-typed (the latter can be similar-typed of Object?).

In our field, we often enough have to deal with unspecified numbers of similar data sources (e.g. the currents of N power converters), which would be nice to handle as a typed list. Also, sometimes these sources are equivalent and sometimes they are not, hence I suggested adding an option to process a Map<K, ObservableValue<...>> instead of just a list - so you can use your own indices, e.g.
```java
zip(mapOf(side->dorosBpms.dorosBeamPosition(ip, beam, plane, side), LEFT, RIGHT)).subscribe(v -> {
                        QuantifiedValue<Double> leftPosition = v.get(LEFT).value();
                        QuantifiedValue<Double> rightPosition = v.get(RIGHT).value();
/// further processing...
                    }, FIRST_UPDATE);
```

Reply:
agree to call it combineLatest :smile: 

for the other proposal, in general I agree but we have to think if we want to clutter the project with func1, 2, 3, 4, 5... or support just 2 max inputs and using BiFunction from java.. what do you think?

Reply:
Ok, probably lets start with Function and BiFunction ... in many cases this shall be ok ... anyhow, functions with that many arguments are not nice ...

Reply:
I agree, both options is good. 
I think I start understanding the maps a bit now ... indeed, having your own indizes can be nice ...but I must admit, I have to check the method signatures still once more ... to see if I find them intuitive... Are they now at the latest proposal state?

Reply:
on the other hand ... probably ossgang commons could be the right place to have Func2 ... Func9... (or similar) ... I used already such functions in tensorics at the latest pull request ... so probably it could be time that tensorics depends on ossgang-commons .... (This might also solve the question of Contextual/ContextAware/Scalar) ... and probably at some point, quantities might go into ossgang-quantities ... and expressions into ossgang-expressions .... at this point then tensorics-expressions and tensorics-quantities might make sense ;-)

Reply:
Yes, this is now the latest state.